src/main/java/com/squareup/javapoet/TypeSpec.java

```diff
@@ -262,7 +274,13 @@ void emit(CodeWriter codeWriter, String enumName, Set<Modifier> implicitModifier
         Collections.<Modifier>emptySet(),
         Collections.<Modifier>emptySet(),
         Collections.<Modifier>emptySet(),
-        Collections.singleton(Modifier.STATIC));
+        Collections.singleton(Modifier.STATIC)),
+
+    ANNOTATION(
```

Comment:
Yeah, I'm sorry. I meant making `Kind` the type public:

```
public enum Kind {
```

It's dumb that we have a public field that's member is a private class.